x-pack/solutions/security/test/security_solution_cypress/config.ts

```diff
@@ -14,7 +14,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     require.resolve('@kbn/test-suites-src/common/config')
   );
   const xpackFunctionalTestsConfig = await readConfigFile(
-    require.resolve('../functional/config.base.js')
+    require.resolve('@kbn/test-suites-xpack/functional/config.base')
```

Comment:
```suggestion
    require.resolve('@kbn/test-suites-xpack-platform/functional/config.base')
```

`x-pack/test` will be deprecated soon